services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -323,6 +344,17 @@ async def create_project(  # pylint: disable=too-many-arguments,too-many-branche
         )
         task_progress.update()
 
+        # 3.2 move project to proper folder
```

Comment:
Question: can something fail here? I do not see anything added in the exception catching below so I ask. normally if something fail the project should not stay in the user list of projects, it should be deleted.

Reply:
Valid point let me double check.

Reply:
Ok so basically as this is a new project id you are inserting, nothing should fail.